docs/business-apps/power-automate/guidance/working-with-get-items-and-get-files.md

```diff
@@ -90,18 +95,15 @@ We support the following query methods and operators.
 * second()
 
 ## Order by query
-
 You can also specify an order by query to order items based off of a column either in ascending or descending order. For example:
```

Comment:
To order items based off of a column either in ascending or descending order, you can also specify an order by query. For example: